.github/unit-tests.yml

```diff
@@ -0,0 +1,25 @@
+name: unit tests
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  unitTest:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - uses: bahmutov/npm-install@v1
```

Comment:
We don’t need this line unless we want a specific npm version, do we?

Reply:
`bahmutov/npminstall` is just for installing dependencies with caching --> works really great. A will just push a new commit with suggested changes